elasticsearch/indexing_unite_legale.py

```diff
@@ -13,7 +13,7 @@ def elasticsearch_doc_generator(data):
     # Serialize the instance into a dictionary so that it can be saved in elasticsearch.
     for index, document in enumerate(data):
         yield ElasticsearchSireneIndex(
-            meta={"id": document["siret_siege"]}, **document
+            meta={"id": document["siren"]}, **document
```

Comment:
Important here. I change the id key of document from siret_siege to siren. It's important because it is this value pivot than I use to update index with new data sources.